packages/charts/react-charting/src/utilities/utilities.ts

```diff
@@ -372,8 +378,9 @@ export function createYAxisForHorizontalBarChartWithAxis(yAxisParams: IYAxisPara
   const tempVal = maxOfYVal || yMinMaxValues.endValue;
   const finalYmax = tempVal > yMaxValue ? tempVal : yMaxValue!;
   const finalYmin = yMinMaxValues.startValue < yMinValue ? 0 : yMinValue!;
+  const maxAbsoluteY = Math.max(Math.abs(finalYmin), Math.abs(finalYmax));
```

Comment:
why making change in horizontal bar chart with axis

Reply:
Will revert